packages/components/nodes/vectorstores/Vectara/Vectara.ts

```diff
@@ -82,16 +82,36 @@ class Vectara_VectorStores implements INode {
                 label: 'Lambda',
                 name: 'lambda',
                 description:
-                    'Improves retrieval accuracy by adjusting the balance (from 0 to 1) between neural search and keyword-based search factors.',
+                    'Enable hybrid search to improve retrieval accuracy by adjusting the balance (from 0 to 1) between neural search and keyword-based search factors.' +
+                    'A value of 0.0 means that only neural search is used, while a value of 1.0 means that only keyword-based search is used. Defaults to 0.0 (neural only).',
+                default: 0.0,
                 type: 'number',
                 additionalParams: true,
                 optional: true
             },
             {
                 label: 'Top K',
                 name: 'topK',
-                description: 'Number of top results to fetch. Defaults to 4',
-                placeholder: '4',
+                description: 'Number of top results to fetch. Defaults to 5',
+                placeholder: '5',
+                type: 'number',
+                additionalParams: true,
+                optional: true
+            },
+            {
+                label: 'MMR K',
+                name: 'mmrK',
+                description: 'Number of top results to fetch for MMR. Defaults to 50',
+                placeholder: '50',
+                type: 'number',
+                additionalParams: true,
+                optional: true
+            },
+            {
+                label: 'MMR diversity bias',
+                name: 'mmrDiversityBias',
+                description: 'The diversity bias to use for MMR. Defaults to 0 (MMR disabled)',
```

Comment:
maybe we can add more details like:
```
The diversity bias to use for MMR, ranging between 0.0 and 1.0. Values closer to 1.0 optimize for the most diverse results. Defaults to 0 (MMR disabled)
```
and specify `step: 0.1`

Reply:
Added. 
note however there is no "step" - this value can be anything b/w 0.0 and 1.0

Reply:
`step` is allow the UI to increment by how many step, by default its 1. So when clicking +, it will increment by 1. By specifying `0.1` we allow smaller increment:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/9dfa42c3-2325-47e4-b02c-3922b597a75a)

Reply:
Oh I see - that makes sense. Added.